Lanchonete/Lanchonete/ Features/ShoppingCart/Presentation/ShoppingCartPresenter.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ShoppingCartPresenter.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/10/21.
+//
+
+import Foundation
+
+public protocol ShoppingCartView: class{
+    func reloadData()
+}
+public class ShoppingCartPresenter {
```

Comment:
This class has no tests.

Reply:
I didn't add tests for the shopping cart since the methods of this class will either use methods of shopping cart manager (that have tests) or override methods of table view. Since I'm on somewhat on a time schedule I think adding tests here is not necessary.